examples/volume_RT_shim_respiratory.py

```diff
@@ -27,32 +13,32 @@
 # > phase = load_niftis( niftiPath ) 5D array (x, y, z, nEcho, nAcq)
```

Comment:
i would comment the "5D array (x, y, z, nEcho, nAcq)", not putting it at the same line as the command